common/js/src/i18n.js

```diff
@@ -0,0 +1,100 @@
+/** @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview This file contains common functionality for
+ * internationalization.
+ */
+
+
+goog.provide('GoogleSmartCard.I18n');
+
+goog.require('GoogleSmartCard.Logging');
+goog.require('goog.log.Logger');
+
+goog.scope(function() {
+
+/** @const */
+var GSC = GoogleSmartCard;
+
+/**
+ * @type {!goog.log.Logger}
+ * @const
+ */
+var logger = GSC.Logging.getScopedLogger('GoogleSmartCard.I18n');
+
+/** @const */
+var I18N_DATA_ATTRIBUTE = 'data-i18n';
+
+/** @const */
+var I18N_DATA_ARIA_LABEL_ATTRIBUTE = 'data-i18n-aria-label';
+
+/**
+ * @param {!Element} element
+ * @param {string} attribute
+ * @param {function(!Element,string)} transformFunction
+ * @private
```

Comment:
No need to do this, all functions are private by default, unless exported explicitly.